assets/js/components/coverage/CoverageIndex.jsx

```diff
@@ -212,27 +124,27 @@ export default (props) => {
             <Mapbox
               data={hotspotStatsData.hotspotStats}
               key="main"
-              followedHotspotStats={
-                followedHotspotStatsData.followedHotspotStats
+              allOrganizationHotspots={
+                allOrganizationHotspotsData.allOrganizationHotspots
               }
               orgHotspotsMap={orgHotspotsMap}
             />
           );
         }
       }
     } else if (currentTab === "followed") {
-      if (followedHotspotStatsData) {
+      if (allOrganizationHotspotsData) {
         if (hotspotAddressSelected) {
           const selectedHotspot =
-            followedHotspotStatsData.followedHotspotStats.filter(
+            allOrganizationHotspotsData.allOrganizationHotspots.filter(
```

Comment:
same here

Reply:
on this one, I can't bc i need more data than what's in `orgHotspotMap`, i.e. coordinates

Reply:
i just realized it's broken even w/ `allOrganizationHotspotsData`

Reply:
got it, feel free to merge